zerver/lib/bugdown/__init__.py

```diff
@@ -1318,6 +1318,31 @@ def __init__(self, parser: Any) -> None:
         super().__init__(parser)
         parser.markdown.tab_length = 4
 
+class BlockQuoteProcessor(markdown.blockprocessors.BlockQuoteProcessor):
+    """ Process BlockQuotes.
+
+        Based on markdown.blockprocessors.BlockQuoteProcessor, but with 2-space indent
+    """
+
+    # Original regex for blockquote is RE = re.compile(r'(^|\n)[ ]{0,3}>[ ]?(.*)')
+    RE = re.compile(r'(^|\n)(?!(?:[ ]{0,3}>\s*(?:$|\n))*(?:$|\n))'
+                    r'[ ]{0,3}>[ ]?(.*)')
+
+    def clean(self, line: str) -> str:
+        # HACK: Silence all the mentions inside blockquotes while
+        # we also remove the '>' from beginning of each line.
+
+        mention_re = re.compile(mention.find_mentions)
+        line = re.sub(mention_re, lambda m: "@_{}".format(m.group('match')), line)
+
+        m = self.RE.match(line)
+        if line.strip() == ">":
+            return ""
+        elif m:
+            return m.group(2)
+        else:
+            return line
```

Comment:
I think this will only work for inline block quotes, and you need something similar to also handle fenced block quotes.

Reply:
I can confirm this works for both:

![image](https://user-images.githubusercontent.com/8033238/50898049-f3578e80-1433-11e9-9136-edabd3b67863.png)

Which makes sense because our `fenced_code` code simply converts each `line` to `> line` and it eventually is parsed by the `BlockQuoteProcessor`.

Reply:
OK, cool, can you add a comment on that?

Reply:
OK, makes sense.